Sources/MongoSwift/WriteConcern.swift

```diff
@@ -3,6 +3,28 @@ import Foundation
 
 /// A class to represent a MongoDB write concern.
 public struct WriteConcern: Codable {
+    /// Majority WriteConcern
+    /// - SeeAlso: https://docs.mongodb.com/manual/reference/write-concern/#writeconcern._dq_majority_dq_
+    // swiftlint:disable force_try
+    public static let majority = try! WriteConcern(w: .majority)
+    // swiftlint:enable force_try
+
+    /**
+     * Returns a customized Majority WriteConcern.
+     *
+     * - Parameters:
+     *   - wtimeoutMS: This option specifies a time limit, in milliseconds, for the write concern.
```

Comment:
the official docs are a little lacking on this one, since they don't actually say what the time limit is for. We should expand on that to say "the maximum amount of time, in milliseconds, that the primary will wait for the write concern to be satisfied before returning a `WriteConcernError`."